authors/apps/articles/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+from django.utils.text import slugify
+from django.db.models.signals import pre_save
+from django.dispatch import receiver
+from authors.apps.core.utils import random_string_generator
+from authors.apps.core.models import TimeModel
+
+
+class Article(TimeModel):
+    ''' This class represents the Article model '''
```

Comment:
Good idea to use double quotes docstring.

Reply:
Let me fix that.